projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -226,8 +249,45 @@ describe('LgCardComponent', () => {
       localFixture.detectChanges();
     });
 
+    it('should set the variant to navigation on the lg-card', () => {
+      expect(component.variant).toContain('navigation');
+    });
+
     it('should set the class lg-card--navigation on the lg-card', () => {
       expect(el.className).toContain('lg-card--navigation');
     });
   });
+
+  describe('when there is orientation', () => {
+    beforeEach(() => {
+      const localFixture = MockRender(`
+        <lg-card [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img imageUrl="/test"></lg-card-hero-img>
+         <lg-card-content>
+            Hello
+         </lg-card-content>
+        </lg-card>
+      `);
+
+      debugElement = localFixture.debugElement;
+      el = debugElement.children[0].nativeElement;
+      component = debugElement.children[0].componentInstance;
+      localFixture.detectChanges();
+    });
+
+    it('should populate the orientationClasses array on the lg-card', () => {
+      expect(component.orientationClasses.length).toBe(3);
+      expect(component.orientationClasses).toContain('lg-orientation--md--horizontal');
```

Comment:
I do test all 3 classes in the test bellow this one, `it('should set the correct orientation classes on the lg-card',` and I think it would be overkill to test them here also, but can add them if needed. :)

Reply:
I agree, let's leave as-is 👍